packages/html/ds/src/paragraph/paragraph.schema.ts

```diff
@@ -0,0 +1,31 @@
+import * as zod from 'zod';
+
+export enum AsEnum {
+  Paragraph = 'p',
+  Span = 'span',
+}
+
+export enum SizeEnum {
+  Large = 'lg',
+  Medium = 'md',
+  Small = 'sm',
+}
+
+export const paragraphSchema = zod.object({
+  content: zod.string({
+    description: 'Content is required',
+    required_error: '',
```

Comment:
I think "Content is required" should go to `required_error` and description should have some info on what that prop is

Reply:
Good catch!